app/models/user.rb

```diff
@@ -32,3 +24,31 @@ def transfer_whittle_savings
     end
   end
 end
+
+
+private 
+
+  def get_user_data
+    # call api to get account info
+    HTTParty.post('http://api119525live.gateway.akana.com:80/user/accounts',
+      :body => {LegalParticipantIdentifier: self.participant_id}.to_json,
+      :headers =>{'Content-Type' => 'application/json', 'Accept' => 'application/json'})
```

Comment:
Add whitespace between hash rocket and nested hash, please! Want to get those style points!